src/generateUserCertificate.ts

```diff
@@ -61,6 +61,14 @@ async function generateuserCertificate ({
       '1.3.6.1.5.5.7.3.1' // id-kp-serverAuth
     ]
   })
+  const attr = pkcs10.attributes as any
+  let dmPubKey = null
+  try {
+    dmPubKey = attr?.[0].values[0].valueBlock.value[1].valueBlock.value[1].valueBlock.value[0].valueBlock.valueHex
+  } catch (e) {
+    console.log(e)
```

Comment:
if we get in this path, the `dmPubKey` will be `null`.
This will make the whole method fail anyway.
I think we need to fail with meanigful error.